### ISSUE TITLE
Distinguish between nsfw and nsfl

### ISSUE THREAD

**Issue by chrisiaut** [voat/voat#451]
Could be implemented with a dialogue after tagging post as nswf, asking if it's also nsfl or with a seperate tag

**Comment by jklepp-tgm** [comment]
IMHO it would be preferable if the tagging system was customizable for each subverse. For some subs a serious or funny tag might be more relevant than a nsfl tag.

**Comment by chrisiaut** [comment]
Great idea! A dynamic system where sub admins can decide what tags make thumbnails dissapear would be great

**Comment by choikwa** [comment]
I always found it a grey area what's NSFW or NSFL. I suggest a sliding scale of "NOT SAFENESS".

**Comment by kayla210** [comment]
I think having a default set of tags like [serious], [nsfw/nsfl], etc. that comes with each subverse would be the best and allow sub admins to designate special tags if they needed. The subcategory for NSFL is a good idea too because I don't think I've ever seen a link that was NSFL but SFW.

**Comment by talons1337** [comment]
>Great idea! A dynamic system where sub admins can decide what tags make thumbnails dissapear would be great

This already exists on both reddit and voat. It's called flair.

**Comment by inkadnb** [comment]
I agree with subverses having their own tag but there should definitely be a default set for all subverses so users don't need to set "No NSFW" for every subverses variation.
